.docker/wp-config.php

```diff
@@ -103,13 +103,14 @@
 define( 'TWITTER_CONSUMER_SECRET_WORDCAMP_CENTRAL', '' );
 define( 'TWITTER_BEARER_TOKEN_WORDCAMP_CENTRAL',    '' );
 
-define( 'WORDCAMP_QBO_APP_TOKEN',       '' );
-define( 'WORDCAMP_QBO_CONSUMER_KEY',    '' );
-define( 'WORDCAMP_QBO_CONSUMER_SECRET', '' );
-define( 'WORDCAMP_QBO_HMAC_KEY',        '' );
+// These can't be empty, but no real values are needed here for local development.
+define( 'WORDCAMP_QBO_APP_TOKEN',       'localtoken' );
+define( 'WORDCAMP_QBO_CONSUMER_KEY',    'localkey' );
+define( 'WORDCAMP_QBO_CONSUMER_SECRET', 'localsecret' );
+define( 'WORDCAMP_QBO_HMAC_KEY',        'localhmac' );
 
-define( 'WORDCAMP_SANDBOX_QBO_CLIENT_ID',        '' );
-define( 'WORDCAMP_SANDBOX_QBO_CLIENT_SECRET',    '' );
+define( 'WORDCAMP_SANDBOX_QBO_CLIENT_ID',        "(Optional) your app's development client ID key goes here" );
+define( 'WORDCAMP_SANDBOX_QBO_CLIENT_SECRET',    "(Optional) your app's development client secret key goes here" );
```

Comment:
I guess the testing instructions are a little convoluted since it's spread between steps 3 and 4. But in Step 3 I'm saying to go to the _Development_ keys tab, not _Production_.

So I'm not suggesting that we use production keys in a local environment.

Reply:
👍